************************
Test -> 3-say_my_name.py
========================

>>> say_my_name = __import__("3-say_my_name").say_my_name

======Succes cases======

>>> say_my_name("Naruto", "Uzumaki")
My name is Naruto Uzumaki

>>> say_my_name("Naruto")
My name is Naruto 

=====Empty cases========

>>> say_my_name("")
Traceback (most recent call last):
...
TypeError: first_name must be a string

Por si acaso

>>> say_my_name(None)
Traceback (most recent call last):
...
TypeError: first_name must be a string

>>> say_my_name(None, "Dulce")
Traceback (most recent call last):
...
TypeError: first_name must be a string


>>> say_my_name("Vino", None)
Traceback (most recent call last):
...
TypeError: last_name must be a string

>>> say_my_name(, "Lee")
Traceback (most recent call last):
...
SyntaxError: invalid syntax

=======Error of types========

>>> say_my_name("Harry", 2)
Traceback (most recent call last):
...
TypeError: last_name must be a string

>>> say_my_name(2, "Harry")
Traceback (most recent call last):
...
TypeError: first_name must be a string

=======Argument Errors=========
>>> say_my_name()
Traceback (most recent call last):
...
TypeError: say_my_name() missing 1 required positional argument: 'first_name'

>>> say_my_name("Martin", "martinez", "andres")
Traceback (most recent call last):
...
TypeError: say_my_name() takes from 1 to 2 positional arguments but 3 were given

>>> say_my_name(, 5)
Traceback (most recent call last):
...
SyntaxError: invalid syntax
